index.js

```diff
@@ -75,6 +77,9 @@ function addAssetsToStream(paths, files) {
 
     // option for newLine in gulp-concat
     if (options.hasOwnProperty('newLine')) {
+        if (options.newLine === ';' && type === 'css') {
+            options.newLine = '';
```

Comment:
Suggest set `options.newLine` to null, as 'gulp-concat' checks type for string and ignores anything else.